client/reader/post-options/index.jsx

```diff
@@ -87,6 +95,27 @@ var PostOptions = React.createClass( {
 		window.open( 'https://wordpress.com/abuse/?report_url=' + encodeURIComponent( this.props.post.URL ), '_blank' );
 	},
 
+	getFollowUrl() {
+		return this.state && this.state.feed ? this.state.feed.get( 'feed_URL' ) : this.props.post.site_URL;
```

Comment:
this can't pull from state if it's used in `getStateFromStores`, as state hasn't been set yet.. Probably should pass the feed object in instead?